lib/galaxy/util/object_wrapper.py

```diff
@@ -23,6 +23,8 @@
     TracebackType,
 )
 
+import sqlalchemy
```

Comment:
I don't think we want sqlalchemy imports in util/ ... maybe move object_wrapper.py into lib/galaxy/security ? Seems like that would be a more appropriate location, given that this is about sanitization ?
Otherwise you could make this a conditional import, but I think lib/galaxy/security is a good home for this.

Reply:
Moving it into security makes perfect sense, will do.